webui/scripts/localization_update.sh

```diff
@@ -10,6 +10,7 @@ echo 'Message lookup ...'
 
 find ../ -regextype posix-egrep -regex '.*(php|phtml)$$' | grep -v vendor | grep -v tests | xargs xgettext --keyword=translate -L PHP --from-code=UTF-8 -o $POTFILE;
 find ../ -regextype posix-egrep -regex '.*(functions.js)$$' | xargs xgettext --keyword=gettext --from-code=UTF-8 -j -o $POTFILE;
+find ../ -regextype posix-egrep -regex '.*(formatter.js)$$' | xargs xgettext --keyword=gettext --from-code=UTF-8 -j -o $POTFILE;
```

Comment:
```suggestion
find ../ -regextype posix-egrep -regex '.*(functions.js|formatter.js)$$' | xargs xgettext --keyword=gettext --from-code=UTF-8 -j -o $POTFILE;
```

Reply:
The two lines can be joined together to avoid duplication

Reply:
that true